lib/config.ts

```diff
@@ -93,9 +93,23 @@ export interface Config {
   sauceKey?: string;
   /**
    * Use sauceAgent if you need customize agent for https connection to
-   * saucelabs.com (i.e. your computer behind corporate proxy)
+   * saucelabs.com (i.e. your computer behind corporate proxy).
+   *
+   * To match sauce agent implementation, use
+   * [HttpProxyAgent](https://github.com/TooTallNate/node-http-proxy-agent)
+   * to generate the agent or use sauceProxy as an alternative where
+   * sauce generates the agent for you based on the proxy. If a sauceProxy is
+   * provided, the sauceAgent will be overridden.
+   */
+  sauceAgent?: HttpProxyAgent;
+  /**
+   * Use sauceProxy that will get generated into an agent for https connection
```

Comment:
Wording is a little awkward. Maybe

"Use sauceProxy to provide the address of a proxy for the connection to saucelabs.com (i.e. your computer is behind a corporate proxy).